### DISCUSSION TITLE
Install Fomantic-UI if installed Semantic-UI before?

### DISCUSSION THREAD

**Discussion by TELLO0815** [fomantic/Fomantic-UI#1203] [General] [answered]
Hi there,

i don't understand on point. If I have already installed Semantic-UI in my project, can I install Fomantic-UI and it will run too? Or must I delete Semantic?

**Comment by y0hami** [comment, marked as answer]
Migrating from Semantic to Fomantic is really easy, since you already have semantic installed (preferably via npm) you can simply change your dependency in your `package.json`

```diff
{
  ...
  "dependencies": {
    ...
-   "semantic-ui": "^2.4.2",
+   "fomantic-ui": "^2.8.2",
    ...
  }
  ...
}
```

Then just run `npm update` and you should be good to go.

If your using a CDN all you need to do is change your CDN link. We have a section on our getting started page with a list of CDN providers. We recommend `jsDelivr` https://fomantic-ui.com/introduction/getting-started.html#using-a-cdn-provider

```diff
- <link rel="stylesheet" type="text/css" href="https://cdn.jsdelivr.net/npm/semantic-ui@2.4.2/dist/semantic.min.css">
+ <link rel="stylesheet" type="text/css" href="https://cdn.jsdelivr.net/npm/fomantic-ui@2.8.2/dist/semantic.min.css">

- <script src="https://cdn.jsdelivr.net/npm/semantic-ui@2.4.2/dist/semantic.min.js"></script>
+ <script src="https://cdn.jsdelivr.net/npm/fomantic-ui@2.8.2/dist/semantic.min.js"></script>
```

Hope this helps, if you have any questions feel free to ask!

**Comment by lubber-de** [comment]
Closing because of no more response. Supposing it worked and no more questions left.
